packages/fileeditor/src/syntaxstatus.tsx

```diff
@@ -46,7 +46,18 @@ namespace EditorSyntaxComponent {
 function EditorSyntaxComponent(
   props: EditorSyntaxComponent.IProps
 ): React.ReactElement<EditorSyntaxComponent.IProps> {
-  return <TextItem source={props.language} onClick={props.handleClick} />;
+  return (
+    <TextItem
+      role="button"
```

Comment:
I could not reach this by keyboard, does it need tabindex?:

```suggestion
      role="button"
      aria-haspopup="true"
      tabIndex={0}
```

Also, does it need `aria-haspopup="true"` as it opens a popup:

![image](https://github.com/user-attachments/assets/62824659-e14c-44d0-b93f-dd655aa20e9d)

Reply:
How can I get this button enabled?